types/networks/src/calibnet/mod.rs

```diff
@@ -84,6 +84,10 @@ pub const HEIGHT_INFOS: [HeightInfo; 17] = [
         height: Height::OhSnap,
         epoch: 682006,
     },
+    HeightInfo {
+        height: Height::Skyr,
+        epoch: 1044660,
```

Comment:
Thinking of using separators in heights one day for more readability, ie `1_044_660`.